### PR TITLE
Fix OCP-19631

### PR DESCRIPTION
**Pull request by xiuwang**
Pass in my local host.
@openshift/devexp-qe @akostadinov Could you help review this? 
Thanks~

### CODE REVIEW

**Comment by wzheng1** [comment]
LGTM,thx

File: features/cli/build.feature [hunk 1]
```diff
@@ -753,6 +753,14 @@ Feature: build 'apps' with CLI
       | Build ConfigMaps:\s+cmtest->. |
       | Build Secrets:\s+secrettest->.|
     And the "ruby-hello-world-1" build completed
+    #Bug 1669981 comment #1, the secreti will not included in the output image.
```
Comment (pruan-rht) [inline features/cli/build.feature]: typo should be `secret` instead of `secreti`

Comment (akostadinov) [reply features/cli/build.feature]: lgtm otherwise

File: features/cli/build.feature [hunk 2]
```diff
@@ -753,6 +753,14 @@ Feature: build 'apps' with CLI
...
+    When I run the :get client command with:
+      | resource      | po                       |
+      | resource_name | ruby-hello-world-1-build |
+      | o             | yaml                     |
+    Then the output should match:
```
Comment (pruan-rht) [inline features/cli/build.feature]: @xiuwang should this line be `Then the output should not match:` because of your comment in line 756?

Comment (xiuwang) [reply features/cli/build.feature]: To adjust comment in line 756, here check secret in the build pod, not in the output image.

**Pull request merged**